properties/P000189.md

```diff
@@ -0,0 +1,15 @@
+---
+uid: P000189
+name: $\sigma$-connected
+refs:
+  - doi: 10.1201/9781315274089
+    name: Continuum Theory
```

Comment:
For books and papers, we usually add the author in parentheses.  I.e., change it to `name: Continuum Theory (Nadler)`.  Makes it easier to figure out what things are when looking at the web page.

Reply:
```suggestion
    name: Continuum Theory (Nadler)
```